packages/clerk-js/src/core/clerk.ts

```diff
@@ -234,7 +233,6 @@ export class Clerk implements ClerkInterface {
   #options: ClerkOptions = {};
   #pageLifecycle: ReturnType<typeof createPageLifecycle> | null = null;
   #touchThrottledUntil = 0;
-  #componentNavigationContext: __internal_ComponentNavigationContext | null = null;
```

Comment:
This is not needed anymore as we have full control to pass the AIOs routing context `navigate` function within `onPendingSession` callback. 

For past context, we previously introduced `#componentNavigationContext` to be able to navigate to tasks within modal routing in which clerk-js didn't have access since it comes from React context.